care/utils/sms/message.py

```diff
@@ -0,0 +1,68 @@
+from typing import TYPE_CHECKING
+
+from django.conf import settings
+
+if TYPE_CHECKING:
+    from care.utils.sms.backend.base import SmsBackendBase
+
+
+class TextMessage:
+    """
+    Represents a text message for transmission to one or more recipients.
+    """
+
+    def __init__(
+        self,
+        content: str = "",
+        sender: str | None = None,
+        recipients: list[str] | None = None,
+        backend: type["SmsBackendBase"] | None = None,
+    ) -> None:
+        """
+        Initialize a TextMessage instance.
+
+        Args:
+            content (str): The message content.
+            sender (Optional[str]): The sender's phone number.
+            recipients (Optional[List[str]]): List of recipient phone numbers.
+            backend (Optional[SmsBackendBase]): Backend for sending the message.
+        """
+        self.content = content
+        self.sender = sender or getattr(settings, "DEFAULT_SMS_SENDER", "")
+        self.recipients = recipients or []
+        self.backend = backend
+
+        if isinstance(self.recipients, str):
+            raise ValueError("Recipients should be a list of phone numbers.")
+
+    def establish_backend(self, fail_silently: bool = False) -> "SmsBackendBase":
```

Comment:
Backend initialisation can happen in init itself, i dont see a usecase where we init and dont sent a message.